plugins/katello/nightly/installation/clients.md

```diff
@@ -68,7 +60,7 @@ Now you are ready to install the client package:
 
 The `katello-host-tools` package reports errata & package profile information, but does not allow you to run remote actions on the clients.
 
-<div class="el5 el6 el7 f29">
+<div class="el6 el7 el8">
```

Comment:
@jturel is `katello-host-tools` even required on el8 for report upload, or is sub-man alone enough these days?

Reply:
Subman covers it all. However host-tools will get installed by katello-host-tools-tracer if folks are using that (not relevant here, just sharing)